src/test/java/posters/pageobjects/components/Pagination.java

```diff
@@ -76,14 +76,14 @@ public void validateElementNumbers(int expectedResultCount)
     @Step("validate visibility right navigation")
     public void validateRightNavigation()
     {
-        paginationComponents.findBy(exactText(Neodymium.localizedText("CategoryPage.pagination.goToNextPage"))).shouldBe(visible);
+        //paginationComponents.findBy(exactText(Neodymium.localizedText("CategoryPage.pagination.goToNextPage"))).shouldBe(visible);
         paginationComponents.findBy(exactText(Neodymium.localizedText("CategoryPage.pagination.goToLastPage"))).shouldBe(visible);
     }
 
     @Step("validate visibility left navigation")
     public void validateLeftNavigation()
     {
-        paginationComponents.findBy(exactText(Neodymium.localizedText("CategoryPage.pagination.goToPrevPage"))).shouldBe(visible);
+        //paginationComponents.findBy(exactText(Neodymium.localizedText("CategoryPage.pagination.goToPrevPage"))).shouldBe(visible);
```

Comment:
please don't keep code as comment, either remove or keep

Reply:
Also if you remove this, please check if the localized test for CategoryPage.pagination.goToPrevPage is still needed, if not please remove it